_sub/security/cloudtrail-config/main.tf

```diff
@@ -8,4 +58,6 @@ resource "aws_cloudtrail" "cloudtrail" {
   include_global_service_events = true
   enable_logging                = true
   enable_log_file_validation    = true
+  cloud_watch_logs_role_arn     = try(aws_iam_role.cloudtrail_to_cloudwatch[0].arn, null)
```

Comment:
CloudWatch is a name, so I think all variables ought to be prefixed with `cloudwatch_` instead of `cloud_watch_`

Reply:
This one is an aws provider resource.